tests/5.1/default/test_default_firstprivate_taskloop.c

```diff
@@ -0,0 +1,55 @@
+//===--- test_default_firstprivate_parallel.c -----------------------------------------------===//
+//
+//  OpenMP API Version 5.1 Aug 2021
+//
+//  This test checks behavior of the default clause when the specified data-sharing-attribute  
+//  is firstprivate. The constructs allowed for a default clause are parallel, teams, task, and taskloop.
+//  This test utilizes the taskloop construct, where firstprivate states that one or more list items are
+//  private to the parallel task, and each of them are initialized with the value of the corresponding
+//  original item. The taskloop construct executes one or more loops in parallel on each thread. 
+//
+////===--------------------------------------------------------------------------------------===//
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int errors;
+
+int test_default_firstprivate_taskloop() {
+  int scalar_var = 6;
+  int arr[N];
+  int sum =0;
+  for (int i=0; i<N; i++){
+        arr[i] = i;
+        sum += arr[i];
+ }
+   #pragma omp taskloop default(firstprivate)
```

Comment:
The way it is right now, it will run sequentially as there are no parallel threads to execute the independent iterations.

Add before line 30: 
#pragma omp parallel num_threads(OMPVV_HOST_NUM_THREADS) //check env var, I am not 100% sure
#pragma omp single